tensorflow/tools/api/golden/tensorflow.AggregationMethod.pbtxt

```diff
@@ -0,0 +1,22 @@
+path: "tensorflow.AggregationMethod"
+tf_class {
+  name: "AggregationMethod"
+  is_instance: "<class \'tensorflow.python.ops.gradients_impl.AggregationMethod\'>"
```

Comment:
Is this class name relevant? The important thing is that it `isinstance` `tensorflow.AggregationMethod`, not so much that it's in gradients_impl...